util/util.go

```diff
@@ -1,6 +1,16 @@
 package util
 
-import "github.com/parquet-go/parquet-go"
+import (
+	"context"
+	"unsafe"
+
+	"github.com/parquet-go/parquet-go"
+	"github.com/thanos-io/objstore"
+)
+
+func YoloString(buf []byte) string {
+	return *((*string)(unsafe.Pointer(&buf)))
```

Comment:
Maybe `string(buf)` instead? 

I get the benefits of allocations but perhaps we want to keep these optimizations for a later stage?

Reply:
yeah.. i will remove :D

Hahaha this is me doing premature optimizations! :P